routing/probability.go

```diff
@@ -0,0 +1,53 @@
+package routing
+
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+// Estimator estimates the probability to reach a node.
+type Estimator interface {
+	getPairProbability(now time.Time, results NodeResults,
+		toNode route.Vertex, amt lnwire.MilliSatoshi) float64
+
+	getLocalPairProbability(now time.Time, results NodeResults,
+		toNode route.Vertex) float64
+}
+
+// ProbabilityEstimatorCfg contains configuration for our probability estimator.
+type ProbabilityEstimatorCfg struct {
```

Comment:
Why move this struct? I think it pertains to a specific probability estimator implementation?